wagtail/admin/templates/wagtailadmin/workflows/index.html

```diff
@@ -3,15 +3,15 @@
 
 {% block content %}
     <header role="banner">
-        <div class="row nice-padding">
+        <div class="row">
             <div class="left">
-                <div class="col header-title">
-                    <h1>{% if view.header_icon %}{% icon name=view.header_icon class_name="header-title-icon" %}{% endif %}
+                <div class="col">
+                    <h1>{% if view.header_icon %}{% icon name=view.header_icon %}{% endif %}
                         {{ view.page_title }}{% if view.subtitle %} <span>{{ view.subtitle }}</span>{% endif %}</h1>
                 </div>
             </div>
             <div class="right">
-                <a href="{% url "wagtailadmin_workflows:index" %}?show_disabled={% if showing_disabled %}false{% else %}true{% endif %}" class="button button-nobg button--icon">
+                <a href="{% url "wagtailadmin_workflows:index" %}?show_disabled={% if showing_disabled %}false{% else %}true{% endif %}" class="button button-secondary button--icon w-inline-flex w-items-center w-gap-2">
```

Comment:
@thibaudcolas

I'm not liking the way we're mixing Tailwind and regular CSS at the moment! We're using both styles together in the .scss files, and also adding tailwind directly to HTML tags. This here is a case where extra Tailwind classes were added directly on a button in one file, but not this file causing the icon to be misaligned.

Is this new approach a permanent decision or just a stop-gap so we can get this out quickly?